GCI Add Model To Model Zoo

rpalakkal opened the pull request:
added vgg19 keras model to model zoo
export to keras works fine
export to caffe worked once but not working anymore

yashdusing commented:
Hey @rpalakkal ! Since vgg16 is already present in the model zoo, we would encourage you to add some other model to the model zoo

rpalakkal commented:
Hello @yashdusing Just added ZFNet to the model zoo; exporting only works for keras.

rpalakkal commented:
@Ram81 I just removed vgg19 and export functionality works for all platforms(caffe, tensorflow, keras) - I added this to the tested_models.md file. I've also attached the screenshot
![screen shot 2018-11-21 at 2 13 24 pm](https://user-images.githubusercontent.com/19766713/48863523-a3dfd500-ed97-11e8-8adf-7a396610d4df.png)

Ram81 commented on docs/source/tested_models.md:
```diff
@@ -17,6 +17,7 @@
 * DeepYeast [\[Source\]](http://kodu.ut.ee/~leopoldp/2016_DeepYeast/code/caffe_model/)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135425bzkzy)
 * SpeechNet [\[Source\]](https://github.com/pannous/caffe-speech-recognition)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135032ctsho)
 * SENet [\[Source\]](https://github.com/hujie-frank/SENet) [\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck)
+* SENet [\[Source\]](https://github.com/dandxy89/ImageModels/blob/master/ZFNet.ipynb)
```
@rpalakkal can you change the name of the model you kept it `SENet`

rpalakkal replied:
@Ram81 just fixed the typo
